tests/app/shared/issue/assignee/assignee.component.spec.ts

```diff
@@ -0,0 +1,138 @@
+
+  const userService: UserService = new UserService(null, null);
+  userService.currentUser = testStudent;
+  const phaseService: PhaseService = new PhaseService(null, null, null, userService, null);
+  phaseService.currentPhase = Phase.phaseTeamResponse;
+  const issueService = jasmine.createSpyObj('IssueService', ['getLatestIssue', 'updateIssue']);
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        AssigneeComponent
+      ],
+      providers: [
+        UserService, IssueService, ErrorHandlingService, PhaseService, PermissionService
+      ],
+      imports: [
+        FormsModule, MaterialModule, ApolloTestingModule, HttpClientModule, BrowserAnimationsModule
+      ]
+    })
+    .overrideProvider(UserService, { useValue: userService })
+    .overrideProvider(PhaseService, { useValue: phaseService })
+    .overrideProvider(IssueService, { useValue: issueService })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AssigneeComponent);
+    component = fixture.componentInstance;
+
+    component.team = dummyTeam;
+    component.issue = dummyIssue;
+    fixture.detectChanges();
+    debugElement = fixture.debugElement;
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should have a placeholder value of - given no assignees', () => {
+    const matPlaceholderValue: HTMLElement = debugElement.query(By.css('.mat-select-placeholder')).nativeElement;
+    expect(matPlaceholderValue).toBeDefined();
+    expect(matPlaceholderValue.innerText).toEqual('-'); // Placeholder Value
+  });
+
+  it('should be able to open the assignee selector', () => {
+    const matSelect: HTMLElement = debugElement.query(By.css('.mat-select-trigger')).nativeElement;
+
+    // Open the assignee selector
+    matSelect.click();
+    fixture.detectChanges();
+    const matOption: HTMLElement = debugElement.query(By.css('.mat-option')).nativeElement;
+    const inputElement: HTMLElement = debugElement.query(By.css('.mat-select-panel')).nativeElement;
+
+    expect(inputElement.children.length).toBe(dummyTeam.teamMembers.length);
+    expect(matOption.attributes.getNamedItem('aria-selected').value).toEqual('false');
+  });
+
+  it('should emit the issueUpdated event upon closing the MatSelect', () => {
+    spyOn(component.issueUpdated, 'emit');
+    const matSelect: HTMLElement = debugElement.query(By.css('.mat-select-trigger')).nativeElement;
+
+    // Open the assignee selector
+    matSelect.click();
+    fixture.detectChanges();
+
+    // Close the matSelect element and make relevant fake calls.
+    const matSelectElement = debugElement.query(By.css('.mat-select')).nativeElement;
+    issueService.updateIssue.and.callFake(() => of(dummyIssue));
+    matSelectElement.dispatchEvent(new Event('closed'));
+    fixture.detectChanges();
+
+    expect(component.issueUpdated.emit).toHaveBeenCalledTimes(1);
```

Comment:
Instead of only testing that the issue was emitted, we can also test that the emitted issue had the correct `assignees`. 

```suggestion
    expect(component.issueUpdated.emit).toHaveBeenCalledWith(jasmine.objectContaining({assignees: [testStudent.loginId]}));

```

Reply:
👍